src/sushi.sol

```diff
@@ -58,6 +63,7 @@ contract SushiJoin is CropJoin {
 
     MasterChefLike  immutable public masterchef;
     address                   public initialMigrator;
+    address                   public initialRewarder;
     TimelockLike              public timelockOwner;
```

Comment:
I'm confused by the naming here--seems like this is the Timelock itself, so why is the variable called "timelockOwner"?

Reply:
This is the `owner` of the MC contract although I agree the names are a little ugly. I'll make a change.